easybuild/easyconfigs/d/Doxygen/Doxygen-1.8.14-GCCcore-7.2.0.eb

```diff
@@ -0,0 +1,20 @@
+name = 'Doxygen'
+version = '1.8.14'
+
+homepage = 'http://www.doxygen.org'
+description = """Doxygen is a documentation system for C++, C, Java, Objective-C, Python, 
+ IDL (Corba and Microsoft flavors), Fortran, VHDL, PHP, C#, and to some extent D."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.2.0'}
+
+sources = ['%(namelower)s-%(version)s.src.tar.gz']
+source_urls = ['http://ftp.stack.nl/pub/users/dimitri/']
+
+builddependencies = [
+    ('binutils', '2.29'),
+    ('flex', '2.6.4'),
+    ('Bison', '3.0.4'),
+    ('CMake', '3.10.2')
```

Comment:
@rubendibattista One of our most recent `Doxygen` easyconfigs also has `pkg-config` as build dep...

It's inconsistent with Doxygen 1.8.13, but can you check on this, and if `pkg-config` is indeed required include it here too?

Bonus point if you issue a separate PR to make `pkg-config` a build dep consistently in the Doxygen 1.8.13 easyconfigs... ;)

Reply:
From Doxygen manual, pkg-config is not listed in the requirements: https://www.stack.nl/~dimitri/doxygen/manual/install.html

Am I misreading something?

Reply:
Maybe @JackPerdue can clarify, it seems like he added this (cfr. #4953).

I can't find a reference to the use `pkg-config` anywhere in the build log for `Doxygen`...

Reply:
I'll go ahead and get this merged, and open a PR to remove the `pkg-config` build dep in the other `Doxygen` easyconfig, it's wrong imho...